handlers/gallery.go

```diff
@@ -1,7 +1,60 @@
 package handlers
 
-import "net/http"
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"path/filepath"
+)
+
+type JSONInfo struct {
+	Content string
+}
+
+type JSONPhoto struct {
+	ID                  string
+	Primary             string
+	Secret              string
+	Server              string
+	Farm                int
+	Photos              string
+	Videos              int
+	Title               JSONInfo
+	Description         JSONInfo
+	NeedsInterstitial   int
+	VisibilityCanSeeSet int
+	CountViews          string
+	CountComments       string
+	CanComment          int
+	CateCreate          string
+	CateUpdate          string
+}
+
+type JSONObject struct {
+	Photosets struct {
+		Page     int
+		Pages    int
+		Perpage  int
+		Potal    int
+		Photoset []JSONPhoto
+	}
+}
+
+var albums JSONObject
+
+func init() {
+	path, _ := filepath.Abs("assets/json/flickr.json")
+	file, e := ioutil.ReadFile(path)
```

Comment:
It looks like `downloadJSONfiles` fetches a JSON and writes it to flickr.json, which is then read by the gallery handler.

Then why write the JSON to a file in the first place? Just keep it in memory.